lib/modules/datasource/azure-bicep-resource/schema.ts

```diff
@@ -2,37 +2,35 @@ import { z } from 'zod';
 
 export const BicepResourceVersionIndex = z
   .object({
-    Resources: z.record(
+    resources: z.record(
       z.string(),
       z.object({
-        RelativePath: z.string(),
-        Index: z.number(),
+        $ref: z.string(),
       }),
```

Comment:
As far as we're only concerned with `resources` keys, its scheme should be `z.record(z.string(), z.unknown())`